openapi/tests/openapi_integration/test_vector_config.py

```diff
@@ -142,3 +143,92 @@ def test_retrieve_vector_specific_quantization():
     assert vectors['text']['quantization_config']['scalar']['always_ram']
     assert config['quantization_config']['scalar']['type'] == "int8"
     assert config['quantization_config']['scalar']['quantile'] == 0.5
+
+@pytest.mark.skip(reason="Takes too long for a sanity test")
+def test_disable_indexing():
+    indexed_name = 'test_collection_indexed'
+    unindexed_name = 'test_collection_unindexed'
+    
+    response = request_with_validation(
```

Comment:
On second thought; this should probably be done at the end of the test to properly clean up, asserting can be skipped.

The `drop_collection` function can be used as helper for this: https://github.com/qdrant/qdrant/blob/4b03cf66ab365b82b3a76212d7fb10809b43317d/openapi/tests/openapi_integration/test_vector_config.py#L13